src/components/select/select.test.jsx

```diff
@@ -166,6 +166,13 @@ describe('src/components/select/select', () => {
       render(<Select {...requiredProps} errorText="not valid">{baseListOptionElements}</Select>);
       expect(screen.getByLabelText('Test label')).not.toBeValid();
     });
+
+    it('does not show errorText on first render until blurred', () => {
+      render(<Select {...requiredProps} required >{baseListOptionElements}</Select>);
+      expect(screen.queryByText('Constraints not satisfied')).toBeNull();
+      fireEvent.blur(screen.getByLabelText('Test label'));
```

Comment:
We prefer `userEvent` testing. Something like:

```suggestion
      userEvent.click(screen.getByLabelText('Test label'));
      userEvent.tab();
```

Reply:
Or click the body or something.